less/accessibility.less

```diff
@@ -0,0 +1,4 @@
+*:focus, *:focus:hover { // use * to win specificity race against normalize.less
+  outline-offset: 0;
+  outline: 3px solid #ffbf47;
```

Comment:
Can we drop this colour (or possibly the whole rule) into `variables.less` just in case we want it overridden for a specific app?

Reply:
This is now `@focus-outline`